plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/ConclaveExtension.kt

```diff
@@ -25,9 +26,7 @@ open class ConclaveExtension @Inject constructor(objects: ObjectFactory) {
     val reflectionConfigurationFiles: ConfigurableFileCollection = objects.fileCollection()
     val serializationConfigurationFiles: ConfigurableFileCollection = objects.fileCollection()
 
-    val runtime: Property<RuntimeType> = objects.property(RuntimeType::class.java).convention(RuntimeType.GraalVM)
-    val graalvm = RuntimeType.GraalVM
-    val gramine = RuntimeType.Gramine
+    val runtime: Property<String> = objects.property(String::class.java).convention(RuntimeType.GraalVM.name)
```

Comment:
Why change this to a String? Doesn't Gradle handle the parsing for you?

Reply:
Not that I could tell. The two lines above here (`val graalvm = RuntimeType.GraalVM` and `val gramine = RuntimeType.Gramine`) are what was being referenced by gradle when we specified `runtime = <whatever>`. So no parsing at all so far as I can tell. If you tried to specify something other than those, you got the following error:

```
A problem occurred evaluating project ':enclave'.
> Could not get unknown property 'gramie' for extension 'conclave' of type com.r3.conclave.plugin.enclave.gradle.ConclaveExtension.
```

Reply:
Perhaps add a comment so that someone doesn't revert this back to an enum

Reply:
Done